lib/embedded_ansible/docker_embedded_ansible.rb

```diff
@@ -0,0 +1,194 @@
+class DockerEmbeddedAnsible < EmbeddedAnsible
+  AWX_WEB_PORT = "54321".freeze
+
+  def self.available?
+    require 'docker'
+    Docker.validate_version!
+  rescue RuntimeError
+    false
```

Comment:
Maybe we should log something here in case the error is not what we are expecting?

Reply:
I wanted to avoid logging here because this could get called whenever we do `EmbeddedAnsible.new` which could be quite a lot in production.

I will look into if we can rescue a more specific error though.